public/coffee/ide/pdf/controllers/PdfController.coffee

```diff
@@ -5,8 +5,9 @@ define [
 	"libs/bib-log-parser"
 	"services/log-hints-feedback"
 ], (App, Ace, HumanReadableLogs, BibLogParser) ->
-	App.controller "PdfController", ($scope, $http, ide, $modal, synctex, event_tracking, logHintsFeedback, localStorage) ->
+	AUTO_COMPILE_TIMEOUT = 3000
 
+	App.controller "PdfController", ($scope, $http, ide, $modal, synctex, event_tracking, logHintsFeedback, localStorage) ->
 		# enable per-user containers by default
 		perUserCompile = true
 		autoCompile = true
```

Comment:
I'm a bit worried about overloading the term "auto compile", as we already use it to mean "auto compile on initial page load" (at least AFAICT). I was struggling with alternative naming though